Stratis.Bitcoin/Configuration/NodeSettings.cs

```diff
@@ -368,12 +369,20 @@ public Network GetNetwork()
         private void SetDefaultDataDir(string appName, Network network)
         {
             string directory = null;
-            var home = Environment.GetEnvironmentVariable("HOME");
-            if (!string.IsNullOrEmpty(home))
+
+            if (!RuntimeInformation.IsOSPlatform(OSPlatform.Windows))
             {
-                this.Logger.LogInformation("Using HOME environment variable for initializing application data.");
-                directory = home;
-                directory = Path.Combine(directory, "." + appName.ToLowerInvariant());
+                var home = Environment.GetEnvironmentVariable("HOME");
+                if (!string.IsNullOrEmpty(home))
+                {
+                    this.Logger.LogInformation("Using HOME environment variable for initializing application data.");
+                    directory = home;
```

Comment:
directory = home; this seems like an extra assignment in vain.
why not do 
` directory = Path.Combine(home, "." + appName.ToLowerInvariant());`
same for localAppData below.

Reply:
Just followed the existing code pattern. I agree it's simpler that way. Modified.